### PR TITLE
Driver class for Car and a few minor changes

### LINKED ISSUE #2: Create a driver class for cars

### PR DESCRIPTION
**Pull request by FanindhraThirunagaru**
Closes #2

### CODE REVIEW

**Comment by OkelloSam21** [comment]
link this *Pr* to Issue #2 then I'll merge it. Thanks for your contribution.

**Review by OkelloSam21 — COMMENTED** [review]
please sync your fork becouse there have been changes in the master branch
Propose your changes on a branch from your fork 

Your code is okay

File: Introduction/hello.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class hello{
     public static void main(String[] args) {
-        System.out.print("Hello World"+"\n"+"I am A java beginner\n");
+        System.out.printf("Hello World"+"%n"+"I am A java beginner%n");
```
Comment (OkelloSam21) [inline Introduction/hello.java:3]: there's an error here use ``` println ``` instead of  ``` printf ```

Comment (FanindhraThirunagaru) [reply Introduction/hello.java:3]: %n is a format to print os dependent next line instead of writing fixed according to the owner os.

Comment (FanindhraThirunagaru) [reply Introduction/hello.java:3]: if ln is used it will print %n as a character rather than next line according to os

Comment (OkelloSam21) [reply Introduction/hello.java:3]: Thanks for the insight

Comment (OkelloSam21) [reply Introduction/hello.java:3]: Did you synch your branch before I can merge your Pr?

Comment (FanindhraThirunagaru) [reply Introduction/hello.java:3]: yes, you can merge the PR

**Pull request merged**